contract/proxy-oracle/soroban/RUNBOOK.md

```
@@ -164,23 +173,31 @@ revoked. Inspect on governance: `has_role --account --role`, `list_role --role`,
+`aggregated_latest` for the current aggregate and `aggregated_history` for
+monotonic audit samples. Historical reads are unavailable while the asset is
+manually or automatically blocked, but they do not apply a freshness cutoff.
 
 ## 7. TTL extension
 
 ```bash
-inv --id $RT  -- extend_ttl
-inv --id $GOV -- extend_ttl --caller <ADMIN>    # admin-only
+inv --id $RT  -- extend_ttl --asset '{"Other":"BTC"}'
+inv --id $GOV -- extend_ttl
 ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)